packages/gestalt/src/ModalAlert.js

```diff
@@ -0,0 +1,244 @@
+// @flow strict
+import { type Node } from 'react';
+import Box from './Box.js';
+import Button from './Button.js';
+import Flex from './Flex.js';
+import Heading from './Heading.js';
+import Icon from './Icon.js';
+import IconButton from './IconButton.js';
+import Modal from './Modal.js';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider.js';
+
+type AbstractEventHandler<T: SyntheticEvent<HTMLElement> | Event, U = {||}> = ({|
```

Comment:
This type will need to be inlined, otherwise we get:
![Screen Shot 2022-10-12 at 4 55 36 PM](https://user-images.githubusercontent.com/12059539/195734518-5c35db4e-b3fd-4709-a1b0-2708f9e093a4.png)

```js
  onClick?: ({|
    event:
      | SyntheticMouseEvent<HTMLButtonElement>
      | SyntheticMouseEvent<HTMLAnchorElement>
      | SyntheticKeyboardEvent<HTMLAnchorElement>
      | SyntheticKeyboardEvent<HTMLButtonElement>,
    dangerouslyDisableOnNavigation: () => void,
  |}) => void,
```